exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/impl/conv/DummyFlatten.java

```diff
@@ -32,6 +34,7 @@
 @FunctionTemplate(name = "flatten", scope = FunctionScope.SIMPLE, nulls = NullHandling.NULL_IF_NULL)
 public class DummyFlatten implements DrillSimpleFunc {
 
+  @Param VarCharHolder in;
```

Comment:
Along the same argument as the above:
https://github.com/hsuanyi/incubator-drill/blob/DRILL-4372_For_Review/exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/DrillSqlOperator.java#L86

Addressed.